### PR TITLE
feat(RTC): use sdpSemantics to stay on Plan B for now

### PR DESCRIPTION
**Pull request by saghul**
It's supported in Chrome >= 65.

### CODE REVIEW

File: modules/browser/BrowserCapabilities.js [hunk 1]
```diff
@@ -304,4 +284,45 @@ export default class BrowserCapabilities extends BrowserDetection {
...
+    supportsSdpSemantics() {
+        return this.isChromiumBased() && this._getChromiumBasedVersion() >= 65;
+    }
+
+    /**
+     * Returns the version of a Chromium based browser.
+     *
+     * @returns {Number}
+     */
+    _getChromiumBasedVersion() {
```
Comment (bbaldino) [inline modules/browser/BrowserCapabilities.js]: does this overlap at all with the other work you did for the chromium-based/version stuff?  i thought you had added something that did this

Comment (saghul) [reply modules/browser/BrowserCapabilities.js]: I just extracted it out since I need to use it in 2 places now.

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by saghul** [comment]
Somehow test results don't reflect here, but it's all 💚 : http://ci.jitsi.org/view/pr-testing/job/lib-jitsi-meet-PR-test/1870/

File: modules/browser/BrowserCapabilities.js (hunk 1 above)
Comment (billmorgan123) [reply modules/browser/BrowserCapabilities.js:303]: can you send the latest version of jitsi meet

**Pull request merged**